lab4/main.py

```diff
@@ -0,0 +1,24 @@
+from ship import Ship
+
+def main():
+    ship1 = Ship(15555, "Leonard", 555, "Lui", 55, 2444)
+    ship2 = Ship(27777, "Di", 777, "Devid", 77, 455)
+    ship3 = Ship(32222, "Caprio", 222, "Lui", 22, 99)
+    ship4 = Ship()
```

Comment:
Please add a print() statement to display the information about the created ships.